crates/cli/src/subcommands/project/rust/Cargo._toml

```diff
@@ -9,5 +9,5 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-spacetimedb = "0.10.1"
+spacetimedb = "0.11.0"
```

Comment:
Ah this fetches from crates.io, right? I guess this is one we shouldn't bump until we publish?

Or we should encourage devs to have local overrides for upcoming versions? (Since presumably you can't use updated codegen while pointing to old libraries?)

Reply:
Yup, I would say let's not bump those until we publish.